conf/application.conf

```diff
@@ -5,3 +5,15 @@ googleAuth.domain = "guardian.co.uk"
 pidfile.path="/dev/null"
 
 play.http.parser.maxMemoryBuffer=4MB
+
+athena {
+  default {
+    driver="com.simba.athena.jdbc.Driver"
+    url="jdbc:awsathena://AwsRegion=eu-west-1"
+    readOnly="true"
+    S3OutputLocation="s3://gu-support-analytics/support-admin-console/"
+    AwsCredentialsProviderClass="com.simba.athena.amazonaws.auth.profile.ProfileCredentialsProvider"
```

Comment:
TODO - this only works locally. But the `scalikejdbc-athena` library doesn't support different configurations - it can only load config from `conf/application.conf`.
Either:
1. submit a PR to make the conf path configurable
2. use a different library

Reply:
solved by using the aws athena client instead